### DISCUSSION TITLE
peptides quant from DIA NN report.

### DISCUSSION THREAD

**Discussion by QinCedars** [vdemichev/DiaNN#124] [Q&A]
Hello, 
I would like to extract the peptide quant from DIANN report ( the largest TSV file). I only see columns with PG.Quantity /Genes.Quantity, and Precursor.Quantity.  I do have following questions:
1. Does Precursor Quantity mean MS1 quant?
2. Where  Can i find exported peptide Quantity?
3. How does the  Peptide quantity calculated? 
4. Where could I find Ion quant for each peptide? 
5.  Is proteolytic sequence (Proteotypic column in the report) a  unique peptide to entire human proteome? 
Thanks for this useful tool. 
Qin

**Comment by vdemichev** [comment]
Hi Qin,

Please see the DIA-NN manual for more detail on the quantities produced. Precursor.Quantity and Precursor.Normalised are MS2 based. See examples here https://github.com/vdemichev/diann-rpackage on how to calculate peptide quantities. Individual fragment quantities are indeed saved in the final report, and listed separated by ';' - both before and after interference removal. Proteotypic = specific to the protein isoform/protein/gene (depending on 'Protein inference' setting).
